virt/install/preparing-cluster-for-virt.adoc

```diff
@@ -57,6 +47,19 @@ endif::[]
 +
 *Bare metal instances or servers offered by other cloud providers are not supported.*
```

Comment:
Not your change, but is this bold on purpose?

Reply:
Hmm. I'm guessing it was intentional at some point, but it rubs me the wrong way. I will un-bold it.